test/parallel/test-fs-read-stream-inherit.js

```diff
@@ -0,0 +1,173 @@
+var common = require('../common');
```

Comment:
This looks fine, but since you mentioned maybe it's a good idea to add some assertions that options were applied :)